apps/gateway/src/routes/api/v1/chats/handlers/addMessage.ts

```diff
@@ -28,13 +28,12 @@ export const addMessageHandler = factory.createHandlers(
         workspace,
         documentLogUuid,
         messages,
-        providerLogHandler: (log) => {
-          // TODO: Review why this is possibly undefined now
-          queues.defaultQueue.jobs.enqueueCreateProviderLogJob!({
```

Comment:
this was wrong provider logs have to be saved in sync

Reply:
I understand but now the request takes longer. This is the tradeoff no?

Reply:
yup